baybe/kernels/kernels.py

```diff
@@ -0,0 +1,53 @@
+"""Collection of kernels."""
+
+from fractions import Fraction
+from typing import Union
+
+from attrs import define, field
+from attrs.validators import in_
+
+from baybe.kernels.base import Kernel
+
+
+def _convert_nu(nu: Union[str, float]) -> float:
+    """Convert the provided value into a float.
+
+    Since the nu parameter might be provided as a fraction, this function tries to
+    convert the str first to a fraction and the to a str. This also works if the value
+    is directly provided as a non-fraction str like '0.5'.
+    Note that this does not validate whether `nu` is one of the three allowed values of
```

Comment:
not sure why you chose to detailed docstring this valdiator, but to save space at least the last sentence `Note that...` can go as its trivial

Reply:
Agree with Martin but would go even a step further. Will add explanation in my review

Reply:
I have to be honest: The reason is because I am personally not happy with the way we comment on validators, default, converters and so on. I would appreciate full docstrings for those, even if they are not necessary. Might be a personal issue resp. due to me not having the same knowledge about all attrs/cattrs stuff like you. This is also the reason why I did not push this (in addition to this being much more work)
But fully agree: The "Note that...." part can go 👍  Will remove once  @AdrianSosic has finished his work here (or feel free to already remove yourself)

Reply:
Changed it as discussed in chat.